packages/rx_bloc_cli/mason_templates/bricks/lib_change_language/__brick__/lib/l10n/network_lookup/lib_change_language_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nLibChangeLanguageLookup extends I18nLibChangeLanguageLookup {
           bundledI18n = I18nLibChangeLanguage(I18nLibChangeLanguageLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nLibChangeLanguage(I18nLibChangeLanguageLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong